examples/rgbpp/spore/local/4-transfer-spore.ts

```diff
@@ -35,6 +36,9 @@ const transferSpore = async ({ sporeRgbppLockArgs, toBtcAddress, sporeTypeArgs }
     ckbFeeRate: BigInt(5000),
   });
 
+  // Save ckbVirtualTxResult
+  saveCkbVirtualTxResult(ckbVirtualTxResult, '4-transfer-spore');
```

Comment:
It is recommended to rename the example name `4-transfer-spore` to `4-transfer-spore-local`, otherwise there will be duplicate files between the queue and local directories.